cypress/e2e/wallet-connection/connect.test.ts

```diff
@@ -12,6 +12,8 @@ describe.skip('disconnect wallet', () => {
     // Disconnect the wallet
     cy.hardhat().then((hardhat) => cy.contains(hardhat.wallet.address.substring(0, 6)).click())
     cy.get(getTestSelector('wallet-disconnect')).click()
+    cy.get(getTestSelector('wallet-disconnect')).contains('Disconnect')
+    cy.get(getTestSelector('wallet-disconnect')).click()
```

Comment:
(I don't know if you can, but) can this be elided to `.click().contains('Disconnect').click()`?